frontend/notification/notificationService.js

```diff
@@ -54,8 +53,8 @@
 
         service.formatMessage = function formatMessage(notification) {
             var entity_type = notification.entity_type;
-            var mainInst = notification.entity.institution_name;
-            var otherInst = notification.from.institution_name;
+            var mainInst = notification.from.institution_name;
```

Comment:
Think if possible, show the new notifications and the old ones. Maybe check if notification hás a new param to decide what show